inc/azure_macro_utils/macro_utils.h

```diff
@@ -202,6 +202,61 @@ const char* MU_C3(MU_, enumIdentifier,_ToString)(enumIdentifier value)
 
 #define MU_ENUM_VALUE_2(enumIdentifier, value) "", MU_ENUM_TO_STRING_2(enumIdentifier, (value)), (int)(value)
 
+#define CONVERT_ENUM_VALUE(from, to) \
+    case (from): \
+        *enum_value_to = (to); \
+        return 0; \
+
+// This macro declares an enum conversion function that translates a value from one enum to another
+#define MU_DECLARE_CONVERT_ENUM(ENUM_TYPE_FROM, ENUM_TYPE_TO) \
+    int MU_C4(convert_enum_, ENUM_TYPE_FROM, _, ENUM_TYPE_TO)(ENUM_TYPE_FROM enum_value_from, ENUM_TYPE_TO* enum_value_to);
```

Comment:
>MU_C4(convert_enum_, ENUM_TYPE_FROM, _, ENUM_TYPE_TO) [](start = 8, length = 53)

This could be `MU_CONVERT_ENUM(..)`, same below

Reply:
Done.

---
In reply to: [394472708](https://github.com/Azure/azure-macro-utils-c/pull/36#discussion_r394472708) [](ancestors = 394472708)